tests/tests.rs

```diff
@@ -233,14 +233,24 @@ fn compare_generated_header(
             }
         }
 
-        // Overwrite the expectation with actual output.
-        if env::var_os("BINDGEN_OVERWRITE_EXPECTED").is_some() {
-            let mut expectation_file =
-                fs::File::create(looked_at.last().unwrap())?;
-            expectation_file.write_all(actual.as_bytes())?;
+        if let Some(var) = env::var_os("BINDGEN_OVERWRITE_EXPECTED") {
+            if var == "1" {
+                // Overwrite the expectation with actual output.
+                let mut expectation_file =
+                    fs::File::create(looked_at.last().unwrap())?;
+                expectation_file.write_all(actual.as_bytes())?;
+            } else { //usecase: var = "meld" -> You can hand check differences
+                let actual_result_path = PathBuf::from(env::var("OUT_DIR").unwrap()).join("meld_bindings.rs");
```

Comment:
So is the idea to only run this one test at a time? If so, that seems fine, but might be worth documenting.

Alternatively we might want to give this a file name based on the test file.

Reply:
I thought that cargo test will stop until you close meld. It does not and I just found this exact race condition.
I will fix it in a few days.

Reply:
Fixed in the newest commit